### ISSUE TITLE
[RPC] Allow more commands to be run remotely

### ISSUE THREAD

**Issue by ScarletRav3n** [Cog-Creators/Red-DiscordBot#2299]
# Feature request

#### Select the type of feature you are requesting:

- [ ] Cog
- [ ] Command
- [x] API functionality

#### Describe your requested feature

Add more support for RPC. Commands should be able to be run remotely, such as 'ping' for the server/channel selected.

**Comment by ghost** [comment]
I agree.

**Comment by Tobotimus** [comment]
RPC support isn't really complete yet, it's not something which has really been used or properly field-tested. It won't be a focus in 3.1 as I believe there are other, more pressing features with higher user demand. I hope to consider it as a focus for 3.2.

**Comment by laggron42** [comment]
@Tobotimus I started to use it not long ago (and made everything work 2 hours ago), and I feel like the biggest problem with RPC is documentation. What's currently written is not clear and I had to rely on a lot on other sources to know how that stuff works.

I'd like to rewrite those docs with a lot of examples + tools for debugging. If you're going to focus on this for 3.2, should I wait until that release before writing those docs?
